engine/classes/Elgg/EntityIcon.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Elgg;
+
+/**
+ * Entities that support icons should implement this interface
+ */
+interface EntityIcon {
```

Comment:
I realize it is pretty standard practice in PHP land but suffixes interface names with `Interface` kind of irks me...